docs/docs/components.mdx

```diff
@@ -836,6 +836,28 @@ Note: The `feature-dimension` for sequence and sentence features does not have t
     "use_word_boundaries": True
   ```
 
+  **Configuring for incremental training**
+
+  To ensure that `sparse_features` are of fixed size during
+  [incremental training](./command-line-interface.mdx#incremental-training), the
+  component should be configured to account for additional patterns that may be
+  added to the training data in future. To do so, configure the `number_additional_patterns`
+  parameter while training the base model from scratch:
+
+  ```yaml-rasa {3}
+  pipeline:
+  - name: RegexFeaturizer
+    number_additional_patterns: 10
+  ```
+
+  If not configured by the user, the component will account for a minimum of 10 additional
+  patterns and a maximum of twice the number of patterns currently
```

Comment:
what does it mean? it is not clear to me what is the default `number_additional_patterns`

Reply:
Rephrased. Is it clearer now?